recirq/quantum_chess/measurement_utils_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import cirq
+import numpy as np
+
+from recirq.quantum_chess.measurement_utils import pauli_decomposition
+
+a1 = cirq.NamedQubit('a1')
+a2 = cirq.NamedQubit('a2')
+a3 = cirq.NamedQubit('a3')
+b1 = cirq.NamedQubit('b1')
+b2 = cirq.NamedQubit('b2')
+b3 = cirq.NamedQubit('b3')
+c1 = cirq.NamedQubit('c1')
+c2 = cirq.NamedQubit('c2')
+c3 = cirq.NamedQubit('c3')
+
+def test_pauli_decomposition_wrong_inputs():
+    H_not_2d = np.array([[[0.5+0.j,  0. +0.5j],
+                          [0. -0.5j,  0.5+0.j ]]], dtype="object")
+    H_not_square = np.array([[0.5+0.j,  0. +0.5j],
+                             [0. -0.5j ]], dtype="object")
+    H_good = np.array([[0.5+0.j,  0. +0.5j],
+                       [0. -0.5j,  0.5+0.j ]], dtype="object")
+    with pytest.raises(ValueError):
```

Comment:
In addition to splitting these out, it might be good to have a match parameter to make sure the error is correct:

`with pytest.raises(ValueError, match='The list size of qubits is 2'):`

Reply:
Thanks Doug. It's fixed.